src/scripts/main.js

```diff
@@ -1,3 +1,59 @@
 'use strict';
 
-// write code here
+const container = document.querySelector('.container');
+const table = document.querySelector('tbody');
+const appRow = document.querySelector('.append-row');
+const remRow = document.querySelector('.remove-row');
+const appColumn = document.querySelector('.append-column');
+const remColumn = document.querySelector('.remove-column');
+const row = table.rows;
+
+container.addEventListener('click', (event) => {
+  if (event.target === appColumn) {
+    [...row].forEach(elemTd => elemTd.append(document.createElement('td')));
+  };
+
+  if (event.target === remColumn) {
+    [...row].forEach(elemtd => elemtd.lastElementChild.remove());
+  };
+
+  if (event.target === appRow) {
+    table.append(row[0].cloneNode(true));
+  };
+
+  if (event.target === remRow) {
+    table.lastElementChild.remove();
+  };
+
+  if (row.length === 10) {
+    appRow.disabled = true;
+  } else {
+    if (row.length <= 10) {
+      appRow.disabled = false;
+    }
+  };
+
+  if (row.length === 2) {
+    remRow.disabled = true;
+  } else {
+    if (row.length >= 2) {
+      remRow.disabled = false;
+    }
```

Comment:
```suggestion
      remRow.disabled = false;
```
do you need this if?